Subject: [PATCH] Hide database connection data
---
Index: Main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Main.py b/Main.py
--- a/Main.py	(revision c642d48da21cd7ed6d54fd9f61f270acf3320de0)
+++ b/Main.py	(date 1739920646256)
@@ -1,13 +1,16 @@
 from fastapi import FastAPI, HTTPException
 from pydantic import BaseModel
 import oracledb
+from dotenv import load_dotenv
+import os
 
 app = FastAPI()
 
+load_dotenv()  # Load variables from .env
 
-db_user = 'RM560508'
-db_password = '050897'
-db_host = 'oracle.fiap.com.br'
+db_host = os.getenv("DB_HOST")
+db_user = os.getenv("DB_USER")
+db_password = os.getenv("DB_PASSWORD")
 
 try:
     connection = oracledb.connect(user=db_user, password=db_password, host=db_host, sid='orcl')
Index: Readme.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Readme.md b/Readme.md
--- a/Readme.md	(revision c642d48da21cd7ed6d54fd9f61f270acf3320de0)
+++ b/Readme.md	(date 1739921295673)
@@ -1,3 +1,18 @@
 To execute the server run the following command:
 
-`uvicorn Main:app --reload`
\ No newline at end of file
+`uvicorn Main:app --reload`
+
+---
+
+> [!WARNING]
+> ### Database connection data
+> 
+> You must create a `.env` file, if you don't have it, with the database connection data in order to be able connect to the database.
+> 
+> The structure should be like:
+> 
+> ```
+> DB_HOST='your_database_host'
+> DB_USER='your_database_user'
+> DB_PASSWORD='your_secure_password'
+> ```
\ No newline at end of file
Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
new file mode 100644
--- /dev/null	(date 1739920187880)
+++ b/.gitignore	(date 1739920187880)
@@ -0,0 +1,1 @@
+.env
\ No newline at end of file
